crates/pallet-wrapped-token/src/lib.rs

```diff
@@ -0,0 +1,165 @@
+//! A substrate pallet that wraps funds related instance.
```

Comment:
This comment does not provide a sufficient description for this pallet.

Reply:
Edited.